sageodata-apispec.yml

```diff
@@ -2530,20 +2607,86 @@ components:
           type: string
         groundElevation:
           format: int32
+          description: The well's current ground elevation in metres.
           type: integer
         referenceElevation:
           format: double
+          description: The well's current elevation from the reference point in metres.
           type: number
         referencePointType:
+          description: Name of the reference point, e.g. TOC (Top of Casing)
+          enum:
+          - GRD
+          - KB
+          - CAB
+          - TOC
+          - FLR
+          - PBP
+          - REF
+          - STP
+          - SCB
+          - WEL
+          - WLC
+          - WIN
+          - WDC
+          - COL
+          - FLN
```

Comment:
Troy, it will be good to provide a description of what these codes mean. :)
If you know what SA Geodata table this is from, I believe the description would be in there.